lib/view_component/codemods/v3_slot_setters.rb

```diff
@@ -1,28 +1,31 @@
-# Usage (in rails console):
-#
-# Run the codemod:
-#
-#     ViewComponent::Codemods::V3SlotSetters.new.call
-#
-# If your app uses custom paths for views, you can pass them in:
-#
-#     ViewComponent::Codemods::V3SlotSetters.new(
-#       view_path: "../app/views",
-#     ).call
+# frozen_string_literal: true
 
 module ViewComponent
+  # Usage:
+  #
+  # Run via rake task:
+  #
+  #     bin/rails view_component:detect_legacy_slots
+  #     bin/rails view_component:migrate_legacy_slots
+  #
+  # Or run via rails console if you need to pass custom paths:
+  #
+  #     ViewComponent::Codemods::V3SlotSetters.new(
+  #       view_path: Rails.root.join("app/views"),
+  #     ).call
   module Codemods
     class V3SlotSetters
       TEMPLATE_LANGUAGES = %w[erb slim haml].join(",").freeze
       RENDER_REGEX = /render[( ](?<component>\w+(?:::\w+)*)\.new[) ]+(do|\{) \|(?<arg>\w+)\b/ # standard:disable Lint/MixedRegexpCaptureTypes
 
       Suggestion = Struct.new(:file, :line, :message)
 
-      def initialize(view_component_path: [], view_path: [])
-        Zeitwerk::Loader.eager_load_all
+      def initialize(view_component_path: [], view_path: [], migrate: false)
+        Rails.application.eager_load!
```

Comment:
If I remember correctly, I used `Zeitwerk` directly so the codemod works in gems that do no require Rails (this needs to be verified).

Reply:
This will cause issues with apps that don't use `Zeitwerk`. Eg some versions of Rails in the `view_component` test suite will fail. The library is Rails-only so I don't think it should be a problem.